Dockerfile

```diff
@@ -1,9 +1,9 @@
 FROM docker.io/openshift/origin-release:golang-1.13 AS builder
 WORKDIR /go/src/github.com/openshift/cincinnati-operator/
 COPY . .
-RUN go build -mod=vendor -o /tmp/build/cincinnati-operator github.com/openshift/cincinnati-operator/cmd/manager
+RUN go build -mod=vendor -o /tmp/build/updateservice-operator github.com/openshift/cincinnati-operator
```

Comment:
No external consumers, so we can always change this post-pivot, but in #78 I used `update-service-operator` instead of `updateservice-operator` as the binary name.